_data/tutorials.yaml

```diff
@@ -257,3 +257,14 @@ naming-changelog-repartition-topics:
     ksql: disabled
     kstreams: enabled
     kafka: disabled
+
+console-consumer-produer-basic:
```

Comment:
Note spelling error in `producer`.

Also, should it be producer-consumer instead of consumer-producer?

Reply:
ugh! thanks!